tests/ui/sized/ensure-overriding-bindings-in-pattern-with-ty-err-doesnt-ice.stderr

```diff
@@ -24,7 +24,9 @@ LL -     let str::<{fn str() { let str::T>>::as_bytes; }}, T>::as_bytes;
 LL +     let str::as_bytes;
    |
 
-error[E0533]: expected unit struct, unit variant or constant, found associated function `str<, T>::as_bytes`
+error[E0533]: expected unit struct, unit variant or constant, found associated function `str<{
+                      fn str() { let (/*ERROR*/); }
```

Comment:
Interestingly this seems to be the first time we leak `pp`'s `/*ERROR*/` in error messages.
At least according to `rg -F '/*ERROR*/' -g 'tests/**/*.stderr'`.

Reply:
For follow-up: Would it make sense to only print function signatures in `const` items in types instead of including their body? What is the output with

```rust
fn main() {
let str::<{fn str<T>() { let _; }}, T>::as_bytes;
}
```